portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetView.testcase

```diff
@@ -623,4 +623,49 @@ definition {
 		}
 	}
 
+	@description = "LPS-189969 - Verify if when the user changes the name of one view, the table will be sorted by the names"
+	@priority = 3
+	test SortViewsByName {
```

Comment:
The purpose of this test is to check that the views are ordered by modified date, from newest to oldest. The name of the test seems like we are testing something different. Please, change the name of the test to something like "ViewsSortedByModifiedDate"